python-rhsm/src/rhsm/certificate2.py

```diff
@@ -82,7 +82,7 @@ def create_from_pem(self, pem, path=None):
 
     def _read_x509(self, x509, path, pem):
```

Comment:
Notice this method signature. It's designed to be able to accept `path=None`, so we should handle that more gracefully (I think maybe something like "Error loading certificate from string" in that case).